commands/inspect_builder.go

```diff
@@ -48,60 +53,64 @@ func InspectBuilder(logger *logging.Logger, cfg *config.Config, client PackClien
 	return cmd
 }
 
-func inspectBuilderOutput(logger *logging.Logger, client PackClient, imageName string, local bool) {
+func inspectBuilderOutput(logger logging.Logger, client PackClient, imageName string, local bool) {
 	info, err := client.InspectBuilder(imageName, local)
 	if err != nil {
 		logger.Error(err.Error())
 		return
 	}
 
 	if info == nil {
-		logger.Info("\nNot present")
+		logger.Info("")
```

Comment:
Not opposed to using control characters instead of separate calls, but it's a matter of personal preference. I wonder if we should be consistent throughout our codebase though.